src/ioctls/vcpu.rs

```diff
@@ -972,6 +972,29 @@ impl VcpuFd {
         Ok(())
     }
 
+    /// Returns the value of the specified vCPU register.
+    ///
+    /// The id of the register is encoded as specified in the kernel documentation
+    /// for `KVM_SET_ONE_REG`.
+    ///
+    /// # Arguments
+    ///
+    /// * `reg_id` - ID of the register.
+    ///
+    pub fn get_one_reg(&self, reg_id: u64) -> Result<u64> {
```

Comment:
```suggestion
    #[cfg(any(target_arch = "arm", target_arch = "aarch64"))]
    pub fn get_one_reg(&self, reg_id: u64) -> Result<u64> {
```

Reply:
Thx.

Reply:
No big deal, but this is not an `aarch64` ioctl.

Reply:
The set_one_reg implementation is also aarch64 only. I will followup with another PR and fix both of them.

Reply:
Both `GET_ONE_REG` and `SET_ONE_REG` are available for all archs.

Reply:
KVM_GET_ONE_REG/KVM_SET_ONE_REG are defined for all archs, but they are gated by KVM_CAP_ONE_REG. So no special treatment needed for x86.